server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -135,7 +134,8 @@ public Invocations getInvocations() {
    }
 
    private CompletionStage<RestResponse> stop(RestRequest restRequest) {
-      invocationHelper.getScheduledExecutor().schedule(() -> invocationHelper.getServer().stop(), SHUTDOWN_DELAY_SECONDS, TimeUnit.SECONDS);
+      List<String> servers = restRequest.parameters().get("server");
```

Comment:
This should be moved to the ```ClusterResource```. The ```ServerResource``` is only about the server where the REST call is being made. It feels strange to call ```http://serverA:11222/rest/v2/server/stop?server=serverB```

Reply:
It has a compilation error now